pipeline_dp/dataset_histograms/histograms.py

```diff
@@ -66,7 +77,15 @@ class HistogramType(enum.Enum):
 
 @dataclass
 class Histogram:
-    """Represents a histogram over integers."""
+    """Represents a histogram over integers.
+
+    To calculate range of the histogram (i.e. min and max values on axis X)
+    you can do the following:
+      * if it is an int histogram, min is 1 and max is bins[-1].upper (or
```

Comment:
Is it hard to introduce lower/upper field to the histogram? (for count lower = 1, upper = None, for sum lower and upper based on computation)

Reply:
what do you think about making them properties of histogram class that are calculated? I think it is a clean way and the easiest one. If we want to make them proper fields and initialize during creation then we need to change `_convert_frequency_bins_into_histogram` method and duplicate there is_integer method because we need to determine the type of histogram (int or float).

Reply:
Can we use the same logic and to create fields? It would be nice to have fields no properties, since we like to convert those dataclasses to protos.

Reply:
done, wdyt?

Reply:
if lgtm in general, I will polish the comments and add tests.

Reply:
LGTM, thanks!